src/main/java/clickup/ui/entities/Space.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package clickup.ui.entities;
+
+/**
+ * Context class.
```

Comment:
Please change the documentation according to our Code Conventions and add an empty line.

Reply:
Done.